src/notebook.cpp

```diff
@@ -122,6 +122,13 @@ class mKCal::Notebook::Private
     QString mSyncProfile;
     QDateTime mCreationDate;
     QHash<QByteArray, QString> mCustomProperties;
+
+    void setModified()
+    {
+        mModifiedDate = QDateTime::currentDateTimeUtc();
+        const QTime at(mModifiedDate.time());
+        mModifiedDate.setTime(QTime(at.hour(), at.minute(), at.second()));
```

Comment:
Could do the extra precision removal on setModifiedDate() where it would apply also calls coming externally? Also the commit talks about microsecond while the rounding is about milliseconds.

Reply:
Indeed, I've drop the millisecond precision for `lastModified`, `created` and `syncDate`. I've reworded the commit to deal with milliseconds and not microseconds.